src/scriptengine/mod_test.rs

```diff
@@ -171,3 +183,18 @@ fn invoke_generic_runner_error() {
 
     assert!(output);
 }
+
+#[test]
+fn invoke_shebang_when_runner_not_specified() {
+    let mut task = Task::new();
+    task.script_runner = None;
+    task.script_extension = Some("py".to_string());
+    task.script = Some(vec![
+        "#!/usr/bin/env python".to_string(),
```

Comment:
if python is not available it will fail.
lets change to linux only test and add here user sh and not py